swagger/v1/component_schemas/NpqApplicationAttributes.yml

```diff
@@ -8,7 +8,6 @@ required:
   - email_validated
   - teacher_reference_number
   - teacher_reference_number_validated
-  - school_urn
```

Comment:
Is this not a required attribute? Without it, how could we (or the provider) know which school the participant is in?